src/nft/utils/listNfts.ts

```diff
@@ -40,7 +41,7 @@ export const ListingMarkets: ListingMarket[] = [
   },
   {
     name: 'OpenSea',
-    fee: 2.5,
+    fee: 0,
```

Comment:
maybe make this a const since they are probably going to change it lmao

Reply:
This is basically the const for the OS ListingMarket object, so changing it here propagated throughout the UX, the only exception is not creating the OS fee consideration item.